main.go

```diff
@@ -43,10 +42,8 @@ func Refresh(ctx context.Context) error {
 
 	posts := fetchAll(ctx, feeds)
 
-	var output bytes.Buffer
-	foo := bufio.NewWriter(&output)
-	renderHtml(foo, posts, "Jan 2006")
-	foo.Flush()
+	output := bytes.NewBuffer([]byte{})
```

Comment:
This can probably be simplified to `output := &bytes.Buffer{}`

Reply:
This works though `		Body:        aws.ReadSeekCloser(output),` doesn't! 

    NotImplemented: A header you provided implies functionality that is not implemented

https://s.natalian.org/2019-01-09/NotImplement.mp4

Any ideas? Seems like a [AWS SDK v2](https://godoc.org/github.com/aws/aws-sdk-go-v2/service/s3) issue.

Reply:
Ah that makes sense. I saw that when inspecting the AWS code and was wondering why they provided their own wrapper around `ReadSeekCloser` when it doesn't really do anything other than proxy the calls down to the underlying object. Looking at the docs, they mention that ReadSeekCloser is actually deprecated in favor of s3manager.Uploader? https://godoc.org/github.com/aws/aws-sdk-go/aws#ReadSeekCloser

Reply:
s3manager.Uploader for *to stream content* tho, right? 

I don't feel like we are streaming since we have the payload in *output*

Reply:
Whoa, that worked! I wonder if s3manager adds much baggage. I am a little confused as to which method should I be using to upload to S3 now. :thinking:

Reply:
Btw, I've switched it to use gocloud.dev/blob/s3blob !